nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-parameter-contexts.js

```diff
@@ -2397,8 +2397,6 @@
          * @param parameterToSelect   Optional, name of the parameter to select in the table.
          */
         showParameterContext: function (id, readOnly, parameterToSelect) {
-            parameterCount = 0;
```

Comment:
I have tested this and it works as advertised.... but I can't figure out why this works or what the original issue is...

Reply:
By not resetting this count each time the parameter context table is displayed the count now grows each time the dialog is opened.

Reply:
@scottyaslan sorry for the late reply, the first fix idea is referenced from variable handling..., but it seems to have the potential issue as you mentioned. maybe we can also fix it too.
I'm reviewing it again, and currently, I find there is a better way to solve it. I'll update later after testing